gemfiles/rails42.gemfile

```diff
@@ -1,6 +1,7 @@
 source 'https://rubygems.org'
 
 gem 'rails', '~> 4.2.0'
+gem 'nokogiri', '~> 1.6.8.1'
```

Comment:
14 days ago, a new version of nokogiri (1.7) has been released (https://github.com/sparklemotion/nokogiri/commit/c5e8e124a5266d0c2779dcd79ced8204d88bf16d). 

This new version does not support a Ruby version <= 2.1 
Here is the error we had previously on Travis: `Gem::InstallError: nokogiri requires Ruby version >= 2.1.0.`.